apps/discovery_streams/lib/discovery-streams/stream/source_handler.ex

```diff
@@ -48,11 +43,6 @@ defmodule DiscoveryStreams.Stream.SourceHandler do
     message["payload"]
   end
 
-  defp log_message(message) do
```

Comment:
lol whoops, i cant believe i was doing this twice.  I blame whoever reviewed my code